### PR TITLE
W-14027157 exchange asset types gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 1]
```diff
@@ -23,14 +23,18 @@ After you upload your policy to Exchange, xref:policies-pdk-apply-policies.adoc[
...
 
 [source,ssh]
 ----
 make publish
 ----
 
-After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appear, similar to the following:
+After publishing the asset, a new `development` state asset appears in Exchange:
+
+image:pdk-dev-asset.png["Development assets appear in Exchange labled DEV."]
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
image:pdk-dev-asset.png["Development assets appear in Exchange labeled DEV."]
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 2]
```diff
@@ -30,7 +32,11 @@ To publish a custom policy, run the `publish` command from the policy's root fol
 make publish
 ----
 
-After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appear, similar to the following:
+After publishing the asset, a new asset labled `DEV` appears in Exchange:
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
After publishing the asset, a new asset labeled `DEV` appears in Exchange:
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 3]
```diff
@@ -53,13 +59,13 @@ To release a custom policy, run the `release` command from the policy's root fol
 make release
 ----
 
-After releasing the asset in Exchange, the asset's GAV appears, similar to the following:
+After releasing the asset in Exchange, the `DEV` label is removed. The asset's GAV appears similar to the following:
```
Comment (NachoRaik) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]: The label is not removed from previously published assets. This is a different asset that is being uploaded.

**Review by NachoRaik — APPROVED** [review]

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 4]
```diff
@@ -13,6 +13,8 @@ Publish a policy if the policy version is still in development. This enables you
 * <<release, Release a Custom Policy>> +
 Release the policy to create a definitive version of the policy. After releasing the policy, you must create a new policy version to edit the policy.
 
+Everytime you publish or release a policy, you create a new `stable` asset in Exchange. For more information about `stable` state assets, see xref:exchange::lifecycle.adoc#stablet-state[Stable State Assets].
```
Comment (valkyrie69) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
Every time you publish or release a policy, you create a new `stable` asset in Exchange. For more information about `stable` state assets, see xref:exchange::lifecycle.adoc#stablet-state[Stable State Assets].
```

File: gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc [hunk 5]
```diff
@@ -30,7 +32,11 @@ To publish a custom policy, run the `publish` command from the policy's root fol
 make publish
 ----
 
-After publishing the asset, a new asset appears in Exchange. The asset's group ID, asset ID, and asset version (GAV) appear, similar to the following:
+After publishing the asset, a new asset labeled `DEV` appears in Exchange:
+
+image:pdk-dev-asset.png["Policy appear in Exchange labeled DEV."]
```
Comment (valkyrie69) [inline gateway/1.5/modules/ROOT/pages/policies-pdk-publish-policies.adoc]:
```suggestion
image:pdk-dev-asset.png["Policy appears in Exchange labeled DEV."]
```

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**